frontend/components/DataTable.tsx

```
@@ -271,40 +337,70 @@ export default function DataTable<T extends RowData>({
+              );
+
+              const menuContent =
+                useEntityContextMenu && entityActionConfig && entityActionContext && onEntityAction ? (
+                  <ContextMenuActions
+                    item={row.original}
+                    selectedItems={selectedRows}
+                    config={entityActionConfig}
+                    actionContext={entityActionContext}
+                    onAction={onEntityAction}
+                    onClearSelection={() => table.toggleAllRowsSelected(false)}
+                  />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)